crates/hc_bundle/src/cli.rs

```diff
@@ -117,7 +121,9 @@ impl HcDnaBundle {
                 crate::init::init_dna(path).await?;
             }
             Self::Pack { path, output } => {
-                let (bundle_path, _) = crate::packing::pack::<DnaManifest>(&path, output).await?;
+                let name = get_dna_name(&path).await?;
+                let (bundle_path, _) =
+                    crate::packing::pack::<DnaManifest>(&path, output, name).await?;
```

Comment:
Kinda weird that we have to read the manifest file twice now, but not a big deal, and I can't think of any immediate suggestion that would be obviously better and not more work than it's work

Reply:
Yeh it's not ideal. Also don't want to do more work than it's work :)